src/System.Runtime.InteropServices.WindowsRuntime/pkg/System.Runtime.InteropServices.WindowsRuntime.pkgproj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Runtime.InteropServices.WindowsRuntime.csproj">
+      <SupportedFramework>net45;dnxcore50;netcore45;wp8;wpa81</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Runtime.InteropServices.WindowsRuntime.builds" />
+
+    <InboxOnTargetFramework Include="net45"/>
+    <InboxOnTargetFramework Include="win8"/>
+    <InboxOnTargetFramework Include="wp80"/>
+    <InboxOnTargetFramework Include="wpa81"/>
+
+    <ExcludeDefaultValidateFramework Include="dnxcore50" />
+
+    <ValidateFramework Include="dnxcore50"> 
```

Comment:
Add a comment that indicates that this is only supported on Windows 8 and higher so we are overriding the validation RIDs.